source/uwp/Renderer/lib/MediaHelpers.cpp

```diff
@@ -224,25 +276,17 @@ namespace AdaptiveCards::Rendering::Uwp::MediaHelpers
                 auto getResourceStreamOperation = resourceResolver.GetResourceStreamAsync(args);
 
                 getResourceStreamOperation.Completed(
-                    [mediaElement, mimeType](winrt::IAsyncOperation<winrt::IRandomAccessStream> operation, winrt::AsyncStatus status) -> void
-                    { return HandleMediaResourceResolverCompleted(operation, status, mediaElement, mimeType); });
+                    [mediaElement, mimeType, adaptiveMedia, renderContext](winrt::IAsyncOperation<winrt::IRandomAccessStream> operation,
+                                                                           winrt::AsyncStatus status) -> void
+                    { return HandleMediaResourceResolverCompleted(operation, status, mediaElement, mimeType, adaptiveMedia, renderContext); });
             }
 
-            mediaElement.MediaOpened(
-                [](winrt::IInspectable const& sender, winrt::RoutedEventArgs const& /*args*/) -> void
+            mediaElement.MediaPlayer().MediaOpened(
+                [](winrt::IInspectable const& sender, winrt::IInspectable const& /*args*/) -> void
                 {
-                    if (const auto mediaElement = sender.try_as<winrt::MediaElement>())
+                    if (const auto mediaPlayer = sender.try_as<winrt::Windows::Media::Playback::MediaPlayer>())
                     {
-                        bool audioOnly = mediaElement.IsAudioOnly();
-                        auto posterSource = mediaElement.PosterSource();
-
-                        if (audioOnly && posterSource == nullptr)
-                        {
-                            // If this is audio only and there's no poster, set the height so
-                            // that the controls are visible.
-                            mediaElement.Height(c_audioHeight);
-                        }
-                        mediaElement.Play();
```

Comment:
The controls show up even if the media is audio only and posterSource is null, so I went ahead and removed this code

Reply:
If that's the case, then we probably don't need `c_audioHeight` - could you please check for usages and delete it if it's the only place where it is used?

Reply:
yep we don't need it, removed!